### DISCUSSION TITLE
Glossary section is not reachable directly from anywhere

### DISCUSSION THREAD

**Discussion by OnkarRuikar** [mdn/mdn-community#662] [Platform] [answered]
The glossary section is reachable only from "See also" links. There needs to be a more explicit link to the glossary section.

a. In the 'References' submenu:
  ![3](https://github.com/mdn/mdn-community/assets/87750369/422ec67b-85e1-4627-8cfb-816a877e9072)
b. suggest more places...

What are your views on this?
Is this location ok? Please suggest other locations that you think would be suitable.

**Comment by OnkarRuikar** [comment]
Following are the comments from the [old discussion](https://github.com/orgs/mdn/discussions/661).
**Dipika:**
I recently proposed internally to add a "?" help menu button at the bottom right corner on the website. The help menu could serve as the home for glossary, contribution guide, and our writing guidelines.

On click, it might look like this:
![mockup](https://private-user-images.githubusercontent.com/23019147/316870744-ed0db2bf-cbd5-4fda-bba6-2b753ccc9d2b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE1NTkxMTMsIm5iZiI6MTcxMTU1ODgxMywicGF0aCI6Ii8yMzAxOTE0Ny8zMTY4NzA3NDQtZWQwZGIyYmYtY2JkNS00ZmRhLWJiYTYtMmI3NTNjY2M5ZDJiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzI3VDE3MDAxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTVmYzYyYjFhNzQ3MjhkMGRkYWM5N2YwZTc4ZDc5YTBmYWU1MzhlY2U2MWJmYWU1NjFlNTQyNWNkMTE5MzAyMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.zZoyJl00vC5s_isrAUF9n-aMRt1RPhJ7fhwcaWGg3u8)
screenshot of proposed help button on MDN

***

**Will:** 
As for the glossary, I don't think it's a big problem that it's not linked. The glossary exists so we have a place to define things in other pages. IMO it's not a particularly big use case for people to decide to browse a glossary. But if we want it, then (3) is reasonable, although I would make the explanation text something like "Glossary of web development terms".

***

**Estelle:**
Suggestion: Add "glossary" to the reference drop down, as per Onkar's screen shot.
Explanation: When I used to look for a link to the glossary, I would always open the references drop-down, as that is where intuitively I felt it would be. Now to get to the glossary, I use the search. I enter "RGB" in the search as it's just 3 characters and I know the first or second result is the glossary term. I think a link from the reference drop-down is more intuitive ;)

**Comment by OnkarRuikar** [comment]
In the community meeting on 27th March following things have been discussed:
a. Dipika gave :+1:  to adding under references menu.
c. Mozilla team to get back later and convey the decision here.

**Comment by dipikabh** [comment]
Hi @OnkarRuikar, we discussed this topic internally. We're leaning towards not adding the Glossary option to the References menu at this time. However, to better understand the existing situation and propose potential improvements, this is now an action item with our UX designer, Anuja, to study how folks use the Glossary on MDN as well as research how similar resources are presented and used on other technical documentation websites. We're hoping this research will help us adopt a more informed strategy/solution. Happy to share that this project is now part of the Q3 goals for Anuja.

As always, we really appreciate you highlighting such concerns and sharing your proposals for improving MDN and user experience.

**Comment by estelle** [comment]
Per Mozilla Discord:
"we're aware the site search needs some work, but that won't be happening until post-platform work this year. For the Glossary - we won't be linking to it in the top nav - but are looking into the footer and/or other places"

**Comment by bsmth** [comment, marked as answer]
The Glossary is linked in the footer now since [the frontend launch](https://developer.mozilla.org/en-US/blog/launching-new-front-end/).
